tests/functional/modules/test_zos_archive_func.py

```diff
@@ -352,12 +351,15 @@ def test_uss_archive_remove_targets(ansible_zos_module, format):
 def test_mvs_archive_single_dataset(ansible_zos_module, format, data_set, record_length, record_format):
     try:
         hosts = ansible_zos_module
+        DATASET = get_dataset(hosts)
```

Comment:
I know I wrote this and now I see it's my fault, but there are things we can correct here :).
Is there a reason for the variable names to be kept in uppercase? Recommend reading https://peps.python.org/pep-0008/#function-and-variable-names and I think is better to keep them as lowercase.

 I was using data_set.get("name") in various places, which is not a descriptive name and although having a variable name DATASET and MVS_DEST_ARCHIVE are more descriptive why not use src_data_set and dest_data_set or archive_data_set?